ecs_fargate/README.md

```diff
@@ -35,67 +35,95 @@ To monitor your ECS Fargate tasks with Datadog, run the Agent as a container in
+    2. For **Image** enter `public.ecr.aws/datadog/agent:latest`.
+    3. For **Env Variables**, add the **Key** `DD_API_KEY` and enter your [Datadog API Key][2] as the value.
+    4. Add another environment variable using the **Key** `ECS_FARGATE` and the value `true`. Click **Add** to add the container.
+    5. Add another environment variable using the **Key** `DD_SITE` and the value {{< region-param key="dd_site" code="true" >}}. This defaults to `datadoghq.com` if you don't set it.
+    6. (Windows Only) Select `C:\` as the working directory.
+11. Add your other application containers to the task definition. For details on collecting integration metrics, see [Integration Setup for ECS Fargate][3].
+12. Click **Create** to create the task definition.
+
+[1]: https://aws.amazon.com/console
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+[3]: http://docs.datadoghq.com/integrations/faq/integration-setup-ecs-fargate
+{{% /tab %}}
+
+{{% tab "AWS CLI" %}}
+##### AWS CLI Task Definition
+
+1. Download [datadog-agent-ecs-fargate][1]. **Note**: If you are using Internet Explorer, this may download as gzip file, which contains the JSON file mentioned below.**
+2. Update the JSON with a `TASK_NAME`, your [Datadog API Key][2], and the appropriate `DD_SITE` ({{< region-param key="dd_site" code="true" >}}). **Note**: The environment variable `ECS_FARGATE` is already set to `"true"`.
+3. Add your other application containers to the task definition. For details on collecting integration metrics, see [Integration Setup for ECS Fargate][3].
 4. Execute the following command to register the ECS task definition:
 
 ```bash
 aws ecs register-task-definition --cli-input-json file://<PATH_TO_FILE>/datadog-agent-ecs-fargate.json
 ```
 
-##### AWS CloudFormation
+[1]: https://docs.datadoghq.com/resources/json/datadog-agent-ecs-fargate.json
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+[3]: http://docs.datadoghq.com/integrations/faq/integration-setup-ecs-fargate
+{{% /tab %}}
+
+{{% tab "CloudFormation" %}}
+##### AWS CloudFormation Task Definition
 
-You can use [AWS CloudFormation][10] templating to configure your Fargate containers. Use the `AWS::ECS::TaskDefinition` resource within your CloudFormation template to set the Amazon ECS task and specify `FARGATE` as the required launch type for that task. 
+You can use [AWS CloudFormation][1] templating to configure your Fargate containers. Use the `AWS::ECS::TaskDefinition` resource within your CloudFormation template to set the Amazon ECS task and specify `FARGATE` as the required launch type for that task.
 
-For example:
+Update this this CloudFormation template below with your [Datadog API Key][2]. As well as include the appropriate `DD_SITE` ({{< region-param key="dd_site" code="true" >}}) environment variable if necessary, as this defaults to `datadoghq.com` if you don't set it.
 
 ```yaml
 Resources:
-  ECSTDNJH3:
+  ECSTaskDefinition:
```

Comment:
did a quick search but why did we use `ECSTDNJH3` in the first place?

Reply:
Probably no specific reason. If you create the CloudFormation template in AWS it will create that resource as `ECSTD` and then 5 random characters like:

![image](https://user-images.githubusercontent.com/15164346/176947190-cb6f17eb-dbef-4f8a-a4a8-b713d6d0c5fc.png)


So probably was just the default resource name brought over. So makes sense to use a normal name, so people aren't wondering what that meant like I was.